packages/core/src/components/Menu/MenuItem/__tests__/MenuItem.test.js

```diff
@@ -67,6 +68,26 @@ describe("<MenuItem />", () => {
     }
   );
 
+  it("should render Label when pass a string", async () => {
+    const labelText = "Label Text";
+    const { getByText } = render(<MenuItem title={title} label={labelText} />);
+    const labelElement = getByText(labelText);
+    await waitFor(() => expect(labelElement).toBeTruthy());
+  });
+
+  it("should render the Label component with props when pass a component", async () => {
+    const labelText = "Label Text";
+    const { getByTestId } = render(
+      <MenuItem title={title} label={<Label text={labelText} color="dark" kind="line" />} />
+    );
+    const labelElement = getByTestId("label");
+    await waitFor(() => expect(labelElement).toBeTruthy());
+    const { getAllByTestId } = within(labelElement);
+    const labelTextElement = getAllByTestId("text")[0];
```

Comment:
does it have more than one test id of text? if so, we should take a look into it, sound like a bug

Reply:
(not in this PR of course)

Reply:
Yes, In the label there are 2 nested text elements, not sure why maybe for the animation?